src/components/forms/TextField/TextField.stories.mdx

```diff
@@ -0,0 +1,32 @@
+import { TextField } from "./TextField";
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import { FormikDecorator } from "../../../story-utils/decorators";
+<Meta
+  title="Pure React Components / Forms / TextField"
+  component={TextField}
+  parameters={{
+    initialValues: {
+      test: 'test'
+    },
+  }}
+/>
+export const Template = (args) => (
+  <TextField {...args} />
+);
+
+<Canvas>
+  <Story
+    name="Default"
+    decorators={[FormikDecorator]}
+    args={{
+      id: 'test',
+      label: "test",
+      description: "This is a custom description",
+      disabled: false,
+      invalid: false,
+      isRequired: true,
+    }}
```

Comment:
this will need the `name` arg/prop as well, matching with the key in the `initialValues` data.